plugins/modules/proxmox_backup_info.py

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2024 Marzieh Raoufnezhad <raoufnezhad at gmail.com>
+# Copyright (c) 2024 Maryam Mayabi <mayabi.ahm at gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = """
+---
+module: proxmox_backup_info
+
+short_description: Retrieve information on Proxmox scheduled backups
+
+description:
+  - Retrieve information such as backup times, VM name, VM ID, mode, backup type, and backup schedule using the Proxmox Server API.
+
+author:
+  - "Marzieh Raoufnezhad (@raoufnezhad) <raoufnezhad@gmail.com>"
+  - "Maryam Mayabi (@mmayabi) <mayabi.ahm@gmail.com>"
+
+options:
+  vm_name:
+    description: The name of the Proxmox VM.
+    required: false
+    type: str
+  vm_id:
+    description: The ID of the Proxmox VM.
+    required: false
+    type: int
+  backup_jobs:
+    description:
+      - The backup_jobs value is true or false.
+      - If set to true, this module just return all backup jobs information.
+      - If set to false, what is listed depends on the other options.
+    required: false
```

Comment:
`required: false` is redundant, please remove it from both here and the argument_spec below.
Also, use semantic markup (such as `V()`), and boolean parameters may have, by definition, only the values `true` and `false`, no need to mention that ;-).
```suggestion
      - Include backup jobs in the result.
      - If V(true), this module just return all backup jobs information.
      - If V(false), the result will not include backup jobs.
```
Or something like that.

Semantic markup is documented in:
https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#semantic-markup-within-module-documentation